crates/graphql/src/schema/objects/nft.rs

```diff
@@ -589,12 +589,12 @@ impl Collection {
             )
             .filter(metadata_collection_keys::collection_address.eq(self.0.mint_address.clone()))
             .filter(metadatas::burned_at.is_null())
-            .filter(metadata_collection_keys::verified.eq(true))
```

Comment:
I think this was an unnecessary check since we are querying for a specific collection, rather than ranking/searching among collections